src/main/java/dk/sdu/se_f22/Main.java

```diff
@@ -1,9 +1,12 @@
 package dk.sdu.se_f22;
 
-import dk.sdu.se_f22.sharedlibrary.db.DBConnection;
-import java.sql.Connection;
+import dk.sdu.se_f22.sharedlibrary.db.LoggingProvider;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LoggingProvider.getLogger(Main.class);
```

Comment:
Should all classes use this instance of the logger? Or should they create their own, with their respective class, that the logging comes from?

Reply:
I think the convention in Log4J is that each class has its own static logger. https://logging.apache.org/log4j/2.x/manual/api.html